mantidproject/mantid
# Reduce ipython notebook header cells

Closes #14903 

These changes move much of the functionality of Mantid header code in ipython notebooks into an imported package. This makes the notebooks look a lot friendlier to users and means that the main content of the notebook is visible as soon as the notebook is opened.

**To Test:**
- Generate an ipython notebook from MantidPlot. For example, load a file and then use the `GenerateIPythonNotebook` algorithm on the resulting workspace.
- Launch ipython notebook (MantidNotebook from start menu in Windows, MantidNotebook launch application on OS X, or `/opt/mantidunstable/bin/mantidpython notebook --notebook-dir=~` on Linux)
- Open the notebook you generated from MantidPlot, it should look like the screenshot below, with only 3 lines of code (plus a couple of comments) in the second cell.
- Run the notebook with Cell->Run All and check all cells are executed (if they ran successfully then they'll have  a number in the square brackets on the left).

![notebook_screen](https://cloud.githubusercontent.com/assets/13455433/12508383/9185b9d4-c0f3-11e5-996b-63703968c9da.png)
